### ISSUE TITLE
TypeError: Cannot read property 'finish_message_type' of null

### ISSUE THREAD

**Issue by lpirola** [nossas/bonde#351]
https://sentry.io/nossas-cidades/bonde-website/issues/211766074/

```
TypeError: Cannot read property 'finish_message_type' of null
  at mapStateToProps (./app/modules/widgets/components/form-finish-message.js:118:34)
...
(45 additional frame(s) were not displayed)
```

**Comment by lpirola** [comment]
This error occured before the fix is published

**Comment by lpirola** [comment]
https://github.com/nossas/bonde-client/commit/1072905c30ee02d112e921bb1c4d18bc8bad814c

Looks like its a bug like these.

**Comment by gabrielrtakeda** [comment]
This will be fixed in v0.4. For now, I will close this issue. If the bug persistis, please open it again.
